api/src/certification/shared/application/helpers/csvHelpers.js

```diff
@@ -0,0 +1,56 @@
+import fs from 'fs';
+
+const { promises } = fs;
+const { readFile, access } = promises;
+
+import papa from 'papaparse';
+
+import { NotFoundError } from '../../../../shared/domain/errors.js';
+import { CsvWithNoSessionDataError } from '../../../session/domain/errors.js';
+
+const optionsWithHeader = {
+  skipEmptyLines: true,
+  header: true,
+  transform: (value, columnName) => {
+    if (typeof value === 'string') {
+      value = value.trim();
+    }
+    if (columnName === '* Sexe (M ou F)') {
```

Comment:
On avait deja noté cela. C'est tres specifique, on ne devrait probablement pas le mettre dans shared (mais peu de risque de collision!)

Reply:
Yes mais le nom de la colonne est tres specifique! mais go ✅